### DISCUSSION TITLE
Allow setting text alignment and background mode via TextEditorState instead of using the toggle method.

### DISCUSSION THREAD

**Discussion by thanglq1** [hm21/pro_image_editor#125] [Ideas]
Hi @hm21. It would be nice to be able to set text alignment, background mode, and fill mode via EditorState instead of using the toggle method.

1. **TextEditor**: Allow setting text alignment (left, center, right) via TextEditorState instead of using the toggle method.
2. **TextEditor**: Allow setting background mode via TextEditorState instead of using the toggle method.
3. **PaintEditor**: The same TextEditor. Allow setting fill mode via PaintingEditorState instead of using the toggle method.

**Comment by hm21** [comment]
This is already possible. Below the examples.

1.
```dart
editorState.align = TextAlign.left;
editorState.setState(() {});
```

2.
```dart
editorState.backgroundColorMode = LayerBackgroundMode.onlyColor;
editorState.setState(() {});
```

3.
```dart
editorState.setFill(true);
editorState.setState(() {});
```

Calling `setState` may not be necessary in some cases, depending on your use case.

**Comment by thanglq1** [comment]
Thank you @hm21. Its working now.
